mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -616,7 +616,10 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             },
         };
 
-        let dex_fee = if repr.dex_fee_burn > MmNumber::default() {
+        let is_kmd = "KMD" == recreate_ctx.maker_coin.ticker() || "KMD" == recreate_ctx.taker_coin.ticker();
+        let dex_fee = if is_kmd {
+            DexFee::Zero
```

Comment:
Q: Is it true that you can skip paying fees if the counterparty is KMD but you are not?

cc @shamardy

Reply:
did I read this correctly.. https://github.com/KomodoPlatform/komodo-defi-framework/issues/2320

Reply:
Yeah, seems like you did it correctly.